src/tasks/__init__.py

```diff
@@ -35,6 +35,7 @@
     "wic",
     "superglue-diagnostic",
 ]
+ALL_DIAGNOSTICS = ["superglue-diagnostic", "winogender-diagnostic", "glue-diagnostic"]
```

Comment:
This'll be easier to maintain if we make it an attribute of the diagnostic tasks themselves, rather than a list.